src/components/app/pageReferrals/yourDistrictRank.tsx

```
@@ -46,15 +47,40 @@ interface YourDistrictRankContentProps {
 function YourDistrictRankContent(props: YourDistrictRankContentProps) {
   const { stateCode, districtNumber } = props
   const countryCode = useCountryCode()
+  const [isUSAddress, setIsUSAddress] = useState<boolean | 'loading'>(false)
 
   const { setAddress, address } = useMutableCurrentUserAddress()
+  const isLoadingAddress = address === 'loading'
 
   const districtRankingResponse = useGetDistrictRank({
     stateCode,
     districtNumber,
   })
 
-  if (districtRankingResponse.isLoading) {
+  const { data } = districtRankingResponse
+
+  const checkIfUSAddress = useCallback(async () => {
+    if (isLoadingAddress || !address || stateCode) return
+
+    setIsUSAddress('loading')
+
+    const addressDetails = await convertGooglePlaceAutoPredictionToAddressSchema(address)
```

Since this is not critical for the UJ (just changing an error message) let's suppress this error and just set the `isUSAddress` variable as `true`